x-pack/plugins/apm/server/routes/backends/route.ts

```diff
@@ -41,7 +40,59 @@ const topBackendsRoute = createApmServerRoute({
   options: {
     tags: ['access:apm'],
   },
-  handler: async (resources) => {
+  handler: async (
+    resources
+  ): Promise<{
+    backends: Array<{
+      currentStats: {
+        latency: {
+          value: number | null;
+          timeseries: Array<import('./../../../typings/timeseries').Coordinate>;
+        };
+        throughput: {
+          value: number | null;
+          timeseries: Array<import('./../../../typings/timeseries').Coordinate>;
+        };
+        errorRate: {
+          value: number | null;
+          timeseries: Array<import('./../../../typings/timeseries').Coordinate>;
+        };
+        totalTime: {
+          value: number | null;
+          timeseries: Array<import('./../../../typings/timeseries').Coordinate>;
```

Comment:
does it make more sense to import `<import('./../../../typings/timeseries').Coordinate>` once and reuse it here?

Reply:
These return types are automatically generated. We could rewrite it to a static import (at the top of the file), but that could lead to naming conflicts, and IMHO this is a good enough for a first attempt. I do want to change it in the long run because a dynamic import takes longer to resolve (presumably becauses it's imported during every type check rather than once at project load)